client/src/components/Header.js

```diff
@@ -1,11 +1,22 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import { Box, Center, HStack, Image, StatusBar, useTheme, Pressable } from 'native-base'
+import { useSelector } from 'react-redux'
 import { useNavigation } from '@react-navigation/native'
 import NotificationIcon from '../assets/icons/Notification'
+import BellIconWithCircle from '../assets/icons/BellWithCircle'
 
 export default () => {
   const { colors } = useTheme()
   const navigation = useNavigation()
+  const { user } = useSelector((state) => state.auth)
+  const { notifications } = user
+  const [unreadNotifications, setUnreadNotifications] = useState([])
```

Comment:
if this is only for selecting bell icon with a circle or not, then no need to keep all status. use boolean instead e.g. `haveUnreadNotification`

Reply:
fixed.